libraries/log_entry.rb

```diff
@@ -17,14 +17,15 @@ class LogEntry < BaseResource
     resource_name :ilo_log_entry
 
     load_base_properties
-    property :log_type, String, required: true, equal_to: ['IEL', 'IML']
+    property :log_type, String, required: true, equal_to: ['IEL', 'IML', :IEL, :IML]
     property :severity_level, String, equal_to: ['OK', 'Warning', 'Critical']
     property :dump_file, String
     property :owner, [String, Integer], default: ENV['USER'] || ENV['USERNAME']
     property :group, [String, Integer], default: ENV['USER'] || ENV['USERNAME']
     property :duration, Integer, default: 24
 
     action :dump do
+      raise 'Please set the :dump_file property (String)' unless dump_file
```

Comment:
why not??  ``` property :dump_file, String, required: true``` Why ain't we leveraging Chef's inbuilt validations if we don't need to put in custom message?